public/doc/swagger-2.yaml

```diff
@@ -230,6 +433,39 @@ paths:
           $ref: '#/responses/404NotFoundResponse'
         '422':
           $ref: '#/responses/422UnprocessableEntityResponse'
+    post:
+      tags:
+        - admins
+      summary: Operate (join or withdraw) a list of approvers to a given group
```

Comment:
My intention is that this endpoint to for any operation to a group. Currently supported operations are join and withdraw. We can expand operations in the future.